### PR TITLE
docs(pipelines): small edits to Managing Pipelines; add info on locking/renaming.

### PR DESCRIPTION
**Pull request by avh121**

### CODE REVIEW

File: guides/user/pipeline/managing-pipelines/index.md [hunk 1]
```diff
@@ -85,14 +85,16 @@ pipeline.
 
 > **Warning**: the JSON you write here is not validated -- Edit as JSON
 essentially allows you to modify the pipeline via a free-form textbox. **Be
-careful!** It is very easy to break the pipeline.
+careful!** It is very easy to break the pipeline, although you can restore from
```
Comment (dorbin) [inline guides/user/pipeline/managing-pipelines/index.md]: s/It is/It's/
https://developers.google.com/style/contractions

Comment (dorbin) [reply guides/user/pipeline/managing-pipelines/index.md]: And feel free to change all such instances throughout. But don't lose any sleep over it.

File: guides/user/pipeline/managing-pipelines/index.md [hunk 2]
```diff
@@ -85,14 +85,16 @@ pipeline.
...
+revision history to roll back any changes.
 
 Spinnaker represents pipelines as JSON behind the scenes. Any changes you make
 to your pipeline using the UI are converted to JSON when Spinnaker saves the
 pipeline.
 
 When you use the **Edit as JSON** feature, you are directly editing the payload.
-This can be useful for working around limitations of the UI.
+**Edit as JSON** allows you to set pipeline fields or properties not exposed by
+the UI.
```
Comment (dorbin) [inline guides/user/pipeline/managing-pipelines/index.md:97]: Yeah, this is a much better way to say this.

File: guides/user/pipeline/managing-pipelines/index.md [hunk 3]
```diff
@@ -109,9 +111,9 @@ Each time you save your pipeline, the current version is added to revision
 history. You can use revision history to diff two versions of a pipeline or to
 restore an older version of a pipeline.
 
-> *Note*: If you are using minio or redis to store your Spinnaker configuration
-files, you won't be able to use revision history because neither minio nor redis
-supports it.
+> *Note*: If you are using minio or redis as your
```
Comment (dorbin) [inline guides/user/pipeline/managing-pipelines/index.md]: s/minio/Minio/
s/redis/Redis/

**Review by dorbin — APPROVED** [review]
Couple a small suggestions. 

LGTM, thanks!

**Comment by demobox** [comment]
Thanks for the update, Nora - looks good to me!

**Pull request merged**